meson.build

```diff
@@ -281,14 +277,13 @@ if libm.found()
   marco_deps += [libm]
 endif
 
-build_sm = not get_option('disable-sm')
-if build_sm and not sm_dep.found()
+build_sm = false
```

Comment:
Why `build_sm = false` ?
In meson_options.txt the option `sm` is enabled as default, or not?

Reply:
`build_sm` is a variable used to display information at the end of the configuration:
```
  ' Session management:       @0@'.format(build_sm),
```

`build_sm` is false by default and is set to true only when dependencies are found:
```
if get_option('sm') and sm_dep.found() and ice_dep.found()
  build_sm = true
endif
```

Reply:
Good.